pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -851,6 +851,7 @@ func (p *MasqueradePodInterface) createBridge() error {
 	bridge := &netlink.Bridge{
 		LinkAttrs: netlink.LinkAttrs{
 			Name: p.bridgeInterfaceName,
+			MTU:  int(p.vif.Mtu),
```

Comment:
I love one liners!

Reply:
Well two lines. Just to be on the safe side. Although from my experiments when the master has an explicit MTU it doesn't take the MTU from a slave, even if the slave has lower value.

Reply:
Does this change solve the issue?
IIRC, the bridge will override the set MTU with the lowest MTU on the enslaved devices, isn't it?

Reply:
ah, that's on masquerade...